external/Makefile

```diff
@@ -107,7 +122,11 @@ ifeq ($(CONFIG_ENABLE_IOTIVITY),y)
 	$(Q) echo "Launching IoTivity Build"
 	$(Q) TOPDIR="$(TOPDIR)" $(TOPDIR)/../external/iotivity/build_iotivity.sh
 endif
+wpa_supplicant_build:
```

Comment:
looks to be something I missed during my cleanup of the patch. Dont think it is needed for anything